extensions/notebook/src/test/model/serverInstance.test.ts

```diff
@@ -38,7 +38,7 @@ describe('Jupyter server instance', function (): void {
 		mockInstall.setup(i => i.outputChannel).returns(() => mockOutputChannel.object);
```

Comment:
You seem to have a lot of changes unrelated to your PR in here. Please get this cleaned up so we don't have to wade through a bunch of other changes when reviewing